src/lib/middleware/userRequiredMiddleware.ts

```diff
@@ -7,6 +7,7 @@ const USER_REQUIRED_ROUTES = [
   "/settings/payments",
   "/settings/shipping",
   "/settings/alerts",
+  "/my-collection",
```

Comment:
Also, shouldn't this be "settings/my-collection" @MrSltun ?

Reply:
Sorry, never mind. I get it now.
We have been using "settings/my-collection" in previous PRs. Do they need updating?
I'm not sure I understand why we no longer want it under /settings/

Reply:
Because deep-linking can still work for `/settings/my-collection`

Reply:
@MrSltun I get it now. Because we don't want to add new deep link support for the settings route on new app versions whereas older versions already support `/my-collection`. 👍 . You might need to pull main before finally merging this. 🙏